checkout/components/user-payment.js

```diff
@@ -60,5 +60,17 @@ app.component('user-payment', {
     expYears() {
       return Array.from({length:10},(v,k)=>k+ new Date().getFullYear())
     }
+  },
+  methods: {
+    changeCardType () {
+      let visa = new RegExp('^4[0-9]{12}(?:[0-9]{3})?$');
+      let mastercard = new RegExp('^(5[1-5]|2[2-7])[0-9]{14}$');
+      let cc = this.paymentDetails.cardNumber;
+      if (visa.test(cc)) {
+        this.paymentDetails.cardType ="visa";
+      } else if (mastercard.test(cc)) {
+        this.paymentDetails.cardType ="master";
```

Comment:
I know there isn't any linting enforced on the project, but the style has been to single quotes.

```suggestion
        this.paymentDetails.cardType = 'visa';
      } else if (mastercard.test(cc)) {
        this.paymentDetails.cardType = 'master';
```